content/manuals/dhi/how-to/scan.md

```diff
@@ -187,27 +187,89 @@ packages and application dependencies.
 After installing Trivy, you can scan a Docker Hardened Image by pulling
 the image and running the scan command:
 
+```console
+$ docker pull dhi.io/<image>:<tag>
+$ trivy image --scanners vuln dhi.io/<image>:<tag>
+```
+
+To filter vulnerabilities using VEX statements, Trivy supports multiple
+approaches. Docker recommeneds using VEXhub, which provides a seamless workflow
```

Comment:
It seems to be referred to as "Vex Hub" in the sources I've seen, eg https://github.com/aquasecurity/vexhub

```suggestion
approaches. Docker recommends using VEX Hub, which provides a seamless workflow
```